spec.html

```diff
@@ -25977,6 +26008,12 @@ <h1>Static Semantics: Early Errors</h1>
         <emu-note>
           <p>The duplicate ExportedNames rule implies that multiple `export default` |ExportDeclaration| items within a |ModuleBody| is a Syntax Error. Additional error conditions relating to conflicting or duplicate declarations are checked during module linking prior to evaluation of a |Module|. If any such errors are detected the |Module| is not evaluated.</p>
         </emu-note>
+
+        <h1>Static Semantics: Early Errors</h1>
+        <emu-grammar>ModuleExportName : StringLiteral</emu-grammar>
+        <ul>
+          <li>It is a Syntax Error if IsStringWellFormedUnicode of the SV of |StringLiteral| is *false*.</li>
```

Comment:
```suggestion
          <li>It is a Syntax Error if IsStringWellFormedUnicode(the SV of |StringLiteral|) is *false*.</li>
```
It's unusual to invoke an AO (IsStringWellFormedUnicode) with the `<name> of <arg>` syntax instead of `<name>(<arg>)`. (And yeah, it's also unusual to have an SDO-invocation as an arg. I'm not sure which is weirder.)